app/webpacker/components/RegistrationsV2/RegistrationAdministration/RegistrationAdministrationTableFooter.jsx

```diff
@@ -5,7 +5,7 @@ import { isoMoneyToHumanReadable } from '../../../lib/helpers/money';
 
 const moneyCountHumanReadable = (registrations, competitionInfo) => {
   const moneyCount = _.sum(registrations.filter(
-    (r) => r.payment.has_paid,
+    (r) => r.payment.payment_amount_iso > 0,
   ).map((r) => r.payment.payment_amount_iso));
```

Comment:
Side note: Can `payment_amount_iso` ever be smaller than zero?

If the backend guarantees that it's between 0 and "something" (considering that due to voluntary donations, there is no real upper bound on reg payments), you might just skip the filter and sum it directly.

I mean, I get your intention, but in the end 30+0+0+30+0 from a few non-paid registraitons mathematically turns out to be the same as a filtered 30+30.

Reply:
Good catch. 

I believe `payment_amount_iso` _could_ be able to be less than 0 - this _shouldn't_ happen, but if for some reason an organizer manages to refund more than the payment amount, `payment_amount_iso` should reflect that (imo).

However, in this case I believe we should also display the _net_ payment amount - ie, the sum of payments including the over-refund - so we shouldn't have any conditions around whether or not to sum the payment amounts. 

Will fix.

Reply:
Ok, looks like my latest push works